Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/service/UserService.java

```diff
@@ -9,34 +9,25 @@
 public class UserService {
     @Autowired
     private UserDAOReal userDAOReal;
-    private  User currentUser;
 
     public boolean addAccount(User user) {
     boolean result=false;
```

Comment:
You don't need this variable

Reply:
this is from past versions. Already deleted